### ISSUE TITLE
Update DNSSEC05

### ISSUE THREAD

**Issue by mattias-p** [dotse/zonemaster#1045]
The next time we make an update to DNSSEC05 we should also fix this:
* The tag NO_RESPONSE_DNSKEY should be renamed to NO_VALID_DNSKEY. This is because the current tag name indicates that no response was received but its semantics also include situations where response with a badly broken DNSKEY is received.

**Comment by matsduf** [comment]
We could find another name, but "NO_VALID_DNSKEY" gives the impression that there are DNSKEY, but no valid ones. When the test case is updated to match the test case template, a new must anyway be given. Or maybe the test case should ignore the non-presence.

**Comment by mattias-p** [comment]
> We could find another name, but "NO_VALID_DNSKEY" gives the impression that there are DNSKEY, but no valid ones. When the test case is updated to match the test case template, a new must anyway be given. Or maybe the test case should ignore the non-presence.

Let's keep the discussion in #1044.

**Comment by matsduf** [comment]
We should have one test case that reports any issue to reach the DNSKEY, not multiple test cases. Preliminary I do not think it should be this test case.

**Comment by matsduf** [comment]
> Another issue I found with this Test Case: it outputs redundant messages because it loops over name servers and instantly outputs messages for each DNSKEY record found. Instead, the results should be grouped by keytag before being outputted. See the specification [here](https://github.com/zonemaster/zonemaster/blob/v2022.2.2/docs/specifications/tests/DNSSEC-TP/dnssec05.md).

Yes, it is a know issue. The specification should be rewritten. When it is rewritten it should be considered what messages it output.
